client/src/app/api/models.ts

```diff
@@ -599,12 +610,24 @@ export interface AnalysisRuleReport extends BaseAnalysisRuleReport {
 
 // Hub type: FileReport - Incidents collated by file
 export interface AnalysisFileReport {
-  issueId: string;
+  issueId: number;
   file: string;
   incidents: number;
   effort: number;
 }
 
+export interface AnalysisIncident {
+  id: number;
+  file: string;
+  line: number;
+  message: string;
+  codeSnip: string;
+  facts: Record<string, string>; // TODO what's actually in here?
+  createTime?: string; // ISO-8601 timestamp
+  createUser?: string;
+  updateUser?: string;
```

Comment:
Maybe we don't need those in the model unless we use them ?

Reply:
Yeah, probably not. I was just putting all the properties in before I started using any, but we probably don't care about these. Will remove